src/cli/domain/get-mocked-plugins.js

```diff
@@ -1,73 +1,77 @@
 const strings = require('../../resources/');
 
-const registerStaticMocks = function(mocks, logger) {
-  return _.map(mocks, (mockedValue, pluginName) => {
-    logger.log(colors.green('├── ' + pluginName + ' () => ' + mockedValue));
+const isMockValid = plugin => {
+  const isFunction = _.isFunction(plugin);
+  const isValidObject =
+    _.isObject(plugin) &&
+    _.isFunction(plugin.register) &&
+    _.isFunction(plugin.execute);
+  return isFunction || isValidObject;
+};
+
+const defaultRegister = (options, dependencies, next) => next();
+
+const registerStaticMocks = (mocks, logger) =>
+  _.map(mocks, (mockedValue, pluginName) => {
+    logger.ok(`├── ${pluginName} () => ${mockedValue}`);
     return {
       name: pluginName,
       register: {
-        register: function(options, dependencies, next) {
-          return next();
-        },
-        execute: function() {
-          return mockedValue;
-        }
+        register: defaultRegister,
+        execute: () => mockedValue
       }
     };
   });
-};
 
-const registerDynamicMocks = function(ocJsonLocation, mocks, logger) {
-  return _.map(mocks, (source, pluginName) => {
-    let p;
+const registerDynamicMocks = (ocJsonLocation, mocks, logger) =>
+  _.map(mocks, (source, pluginName) => {
+    let pluginMock;
     try {
-      p = require(path.resolve(ocJsonLocation, source));
+      pluginMock = require(path.resolve(ocJsonLocation, source));
     } catch (er) {
       logger.err(er.toString());
       return;
     }
 
-    if (!_.isFunction(p)) {
-      logger.err(strings.errors.cli.MOCK_PLUGIN_IS_NOT_A_FUNCTION);
+    if (!isMockValid(pluginMock)) {
+      logger.err(`├── ${pluginName} () => Error (skipping)`);
+      logger.err(strings.errors.cli.MOCK_PLUGIN_IS_NOT_VALID);
       return;
     }
 
-    logger.log(colors.green('├── ' + pluginName + ' () => [Function]'));
+    const register = pluginMock.register || defaultRegister;
+    const execute = pluginMock.execute || pluginMock;
+
+    logger.ok(`├── ${pluginName} () => [Function]`);
+
     return {
       name: pluginName,
-      register: {
-        register: function(options, dependencies, next) {
-          return next();
-        },
-        execute: p
-      }
+      register: { execute, register }
     };
-  }).filter(p => p);
-};
+  }).filter(pluginMock => pluginMock);
 
 const findPath = function(pathToResolve, fileName) {
-  const rootDir = fs.realpathSync('.'),
-    fileToResolve = path.join(pathToResolve, fileName);
+  const rootDir = fs.realpathSync('.');
+  const fileToResolve = path.join(pathToResolve, fileName);
 
   if (!fs.existsSync(fileToResolve)) {
     if (pathToResolve === rootDir) {
       return undefined;
     } else {
-      const getParent = function(x) {
-          return x
-            .split('/')
-            .slice(0, -1)
-            .join('/');
-        },
-        parentDir = pathToResolve ? getParent(pathToResolve) : rootDir;
+      const getParent = x =>
```

Comment:
I think is never too bad to name things for what they are: `pathToResolve` or similar in this case instead of `x`